tests/foreman/ui/test_errata.py

```diff
@@ -467,6 +496,35 @@ def test_positive_content_host_library(session, module_org, vm):
         assert content_host_erratum[0]['Id'] == CUSTOM_REPO_ERRATA_ID
 
 
+@tier3
+def test_positive_content_host_search_type(session, erratatype_vm):
+    """ Search for errata on a content_host by type
+
+    :id: 59e5d6e5-2537-4387-a7d3-637cc4b52d0e
+
+    :Setup: Content Host with applicable security errata
+
+    :Steps: Search for errata on content host by type (ie 'type = security')
+
+    :BZ: 1653293
+
+    :CaseLevel: Integration
+    """
+
+    pkgs = " ".join(FAKE_9_YUM_OUTDATED_PACKAGES)
+    assert _install_client_package(erratatype_vm, pkgs, errata_applicability=True)
+
+    with session:
+        ch_erratum = session.contenthost.search_errata(
+            erratatype_vm.hostname, "type = security", environment='Library Synced Content')
+
+        assert len(ch_erratum) == FAKE_9_YUM_SECURITY_ERRATUM_COUNT
+
+        # check IDs just to be safe
+        errata_ids = sorted([e['Id'] for e in ch_erratum])
+        assert errata_ids == sorted(FAKE_9_YUM_SECURITY_ERRATUM)
```

Comment:
Perhaps make them set for assertion?

```python
        errata_ids = [e['Id'] for e in ch_erratum]
        assert set(errata_ids) == set(FAKE_9_YUM_SECURITY_ERRATUM)
```

It's mostly a personal preference, but I just don't like calling `sorted()` when I don't care about order.

Reply:
Yeah I see your point, but I don't know if replacing them with sets is good in this case. Sets by defininition don't have repeated elements and in general if your trying to validate the data from a UI page your testing you want to preserve as much data about it as possible. 

Although in this test case it would probably not be an issue, I imagine there might in the future be some additions to this test (possibly by someone not as familiar with sets or python) and I want to be as concise as possible